BrowserKit/Sources/Common/Theming/DefaultThemeManager.swift

```diff
@@ -178,7 +178,6 @@ public final class DefaultThemeManager: ThemeManager, Notifiable {
 
     private func fetchSavedThemeType() -> ThemeType {
         if privateModeIsOn { return .privateMode }
-        if nightModeIsOn { return .dark }
```

Comment:
nightMode != darkTheme, because of this line the user default set to dark theme when night mode is on and app started after a dismiss.